hyperlink/_url.py

```diff
@@ -1594,8 +1594,8 @@ class DecodedURL(object):
 
     Args:
        url (URL): A :class:`URL` object to wrap.
-       lazy (bool): Whether to pre-decode all parts of the URL to
-           check for validity. Defaults to True.
+       lazy (bool): Set to True to avoid pre-decode all parts of the
```

Comment:
Minor grammar error here, should now be `avoid pre-decoding`, but the sentence is still a bit awkward compared to the original. Maybe: "If `True`, decode all parts of the URL immediately to check that they are valid. Otherwise when `False`, do so lazily as needed, so an error may be raised <whenever that is true>."